src/mmw/mmw/settings/gwlfe_settings.py

```diff
@@ -451,3 +451,86 @@
     'MonthDays': [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31],
     'WeatherNull': -99999,  # This value is used to indicate NULL in ms_weather dataset
 }
+
+# Maps NLCD + Hygrological Soil Group to a Curve Number
```

Comment:
Consider adding a comment about where these values came from in the original Mapshed code.